internal/service/management_api.go

```diff
@@ -0,0 +1,60 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	metrics "github.com/slok/go-http-metrics/metrics/prometheus"
+	"github.com/slok/go-http-metrics/middleware"
+	"github.com/slok/go-http-metrics/middleware/std"
+	"github.com/topfreegames/maestro/internal/config"
+	"github.com/topfreegames/maestro/internal/core/monitoring"
+	"github.com/topfreegames/maestro/internal/handlers"
+	api "github.com/topfreegames/maestro/protogen/api/v1"
+	"go.uber.org/zap"
+)
+
+func ProvideManagementMux(ctx context.Context, pingHandler *handlers.PingHandler) *runtime.ServeMux {
```

Comment:
The `cmd/management_api` will be the only one using this function. Do you think we can move it there (as this `service` package will be shared between multiple commands)?